homeassistant/components/snapcast/media_player.py

```diff
@@ -149,16 +150,27 @@ class SnapcastGroupDevice(MediaPlayerEntity):
 
     def __init__(self, group, uid_part):
         """Initialize the Snapcast group device."""
+        self._attr_available = True
         self._group = group
+        self._entry_id = None
         self._uid = f"{GROUP_PREFIX}{uid_part}_{self._group.identifier}"
 
     async def async_added_to_hass(self) -> None:
         """Subscribe to group events."""
         self._group.set_callback(self.schedule_update_ha_state)
+        if self.registry_entry:
```

Comment:
The snapcast entities will always have a registry_entry, won't they?
Instead of the if-statement, add an  assert:
```py
# Snapcast entities are always in the entity registry
assert self.registry_entry is not None
```

Reply:
Probably you are right, I don't know these internals so well. Assert is a good idea, will change it.